updating/understanding-openshift-upgrade-duration.adoc

```diff
@@ -0,0 +1,91 @@
+
+[IMPORTANT]
+====
+With {product-title} 4 upgrades, an operating system lifecycle management and maintenance is made as a part of the cluster upgrade.  This only applies to {op-system-first} nodes.
+====
+
+The time to complete this process depends on many factors including the node and infrastructure configuration. This process might take 5 minutes to complete per node.
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../architecture/architecture-installation.html#update-service-overview_architecture-installation[About the OpenShift Update Service]
+
+
+== Factors affecting the upgrade duration
+In addition to Cluster Version Operator (CVO) and Machine Config Operator (MCO), the following factors also impact the upgrade duration:
+
+* Pod Disruption Budget (PDB)
+* Cluster node health
+* Number of nodes in the cluster
+* Red Hat Enterprise Linux (RHEL) nodes
+
+=== Pod disruption budget
+You must consider pod disruption budgets and pod anti-affinity when you configure `maxUnavailable` field in the machine configuration pool.
+
+For example, if there is a cluster with three worker nodes, you must configure the disruption budget such that it has minimum two replicas be always running. When you configure pod anti-affinity, you must specify that two replicas should never run on the same node. To avoid pod scheduling issues, you must ensure the availability of minimum two worker nodes.
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../nodes/pods/nodes-pods-configuring.adoc#nodes-pods-configuring-pod-distruption-about[pod disruption budget]
+* xref:../nodes/scheduling/nodes-scheduler-pod-affinity.adoc#nodes-scheduler-pod-affinity-about[pod anti-affinity]
+
+=== Cluster node health
+An unhealthy node takes longer time to upgrade. It is recommended to resolve any issues before you begin the cluster upgrade.
+
+=== Red Hat Enterprise Linux nodes
+{op-system-base-full} worker nodes require an additional usage of Openshift-ansible to update node binary components. This rolling update is similar to the MCO initiated rolling update and therefore it has a similar duration. The time it takes to reboot a particular node varies significantly. In cloud instances, the reboot takes less than a minute whereas in physical bare metal hosts the reboot takes more than 15 minutes.
+
+== Estimate cluster upgrade time
+To estimate your cluster upgrade time, you can use the following convention:
+
+----
+CVO Target Upgrade Payload Deployment time + (Number of node upgrades x MCO node upgrade time) + (Number of RHEL nodes x RHEL Node Upgrade time) = Cluster upgrade time
+----
+
+For example, a cluster with nine nodes, which has no RHEL nodes and unhealthy nodes, and the worker node set to `maxUnavailable=2` on the machine configuration pool could have a minimum upgrade time of 60 minutes.
```

Comment:
it is not clear from here how we get from 9 nodes to 3 rounds of node upgrades.  I think we should explicitly say something like:

> For example, consider a cluster with the following machine configuration pools:
>
> * `master` with three nodes and `maxUnavailable=1`, which will update in three rounds.
> * `worker` with six nodes and `maxUnavailable=2`, which will update in three rounds.
>
> If the cluster has no RHEL nodes or unhealthy nodes, the minimum update time is:

Reply:
Done.